src/test/scala/uk/gov/hmrc/play/microservice/bootstrap/JsonErrorHandlingSpec.scala

```diff
@@ -16,41 +16,64 @@
 
 package uk.gov.hmrc.play.microservice.bootstrap
 
-import org.scalatest.concurrent.ScalaFutures
+import ch.qos.logback.classic.Level
+import org.mockito.Mockito._
+import org.scalatest.LoneElement
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
 import org.scalatest.mock.MockitoSugar
-import org.scalatest.{Matchers, WordSpecLike}
 import play.api.mvc.RequestHeader
-import play.api.{PlayException, GlobalSettings}
-import uk.gov.hmrc.play.http.{UnauthorizedException, BadRequestException, NotFoundException}
+import play.api.{GlobalSettings, Logger}
+import uk.gov.hmrc.play.http.{BadRequestException, NotFoundException, UnauthorizedException}
+import uk.gov.hmrc.play.test.{LogCapturing, UnitSpec}
 
-class JsonErrorHandlingSpec extends WordSpecLike with Matchers with ScalaFutures with MockitoSugar {
-
-  val jsh = new GlobalSettings with JsonErrorHandling {}
-
-  val requestHeader = mock[RequestHeader]
+class JsonErrorHandlingSpec extends UnitSpec with ScalaFutures with MockitoSugar with LogCapturing with LoneElement with Eventually {
 
   "error handling in onError function" should {
 
-    "convert a NotFoundException to NotFound response" in {
+    "convert a NotFoundException to NotFound response" in new Setup {
       val resultF = jsh.onError(requestHeader, new NotFoundException("test")).futureValue
       resultF.header.status shouldBe 404
     }
```

Comment:
@scottkwalker Whilst you are here, what do you think of adding another check, to each of these existing cases, for the json generated, i.e. for the result of the `Json.toJson(errorResponse)`?  Let me know if that doesn't make sense - I'm only looking at the diffs on the PR.  Thanks.